src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -274,30 +320,34 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
     * or aspect ratio are met. Otherwise, returns a one-item Seq containing
     * only this block.
     */
```

Comment:
The documentation doesn't seem to be up-to-date with the changes that were made - could you add something here that describes the `generation` parameter?